template/src/ontology/run.sh.jinja2

```diff
@@ -11,15 +11,22 @@
 # See README-editors.md for more details.
 
 IMAGE=${IMAGE:-odkfull}
-
+MEMORY_GB=${MEMORY_GB:-8}
+MEMORY_JAVA={% if project.robot_java_args is defined %}{{ project.robot_java_args }}{% else %}-Xmx${MEMORY_GB}G{% endif %}
```

Comment:
Here, the value defined in the configuration file takes precedence from what is specified in the environment. I think it should be the other way round. It should be possible to occasionally override the value from the configuration file with a value from the environmemt.